frontend/src/pages/SearchUISearchPage.js

```diff
@@ -92,7 +93,7 @@ type SearchUISearchPageProps = {
   orderHint: Array<string>;
   /** Controls the colors used to show various entity types (the value can be any valid CSS color) */
   entityColors: Map<string, string>;
-  /**
+  /*
```

Comment:
This was intentionally a JSDoc ( /** */) comment so it appears in the documentation...